randnum.c

```diff
@@ -0,0 +1,32 @@
+/*Sean Kee*/
```

Comment:
PERFECT job! 💯 great comments and great implementation! You did everything well and efficiently! 😃